src/js/components/video/Controls.js

```diff
@@ -75,7 +128,7 @@ export default class Controls extends Component {
             <VideoTime currentTime={currentTime} duration={duration} />
             {this._renderMuteButton()}
             {allowFullScreen ?
-              <VideoFullscreenButton onClick={fullscreen} /> : undefined}
+              <VideoFullscreenButton onClick={fullscreen} /> : null}
```

Comment:
null here, not sure if I'm missing something, but does undefined breaks something here?